src/app/modules/main/module.nim

```diff
@@ -494,9 +494,22 @@ method communityJoined*[T](
   self.communitySectionsModule[community.id].load(events, settingsService, contactsService, chatService, communityService, messageService)
 
   let communitySectionItem = self.createCommunityItem(community)
-  self.view.addItem(self.createCommunityItem(community))
+  self.view.addItem(communitySectionItem)
   self.setActiveSection(communitySectionItem)
 
+method communityLeft*[T](self: Module[T], communityId: string) =
+  if(not self.communitySectionsModule.contains(communityId)):
+    echo "main-module, unexisting community key to leave: ", communityId
+    return
+
+  self.communitySectionsModule.del(communityId)
+
+  self.view.removeItem(communityId)
+
+  if (self.controller.getActiveSectionId() == communityId):
+    # Set active section to the first one instead (should be chat, but doesn't matter)
+    self.setActiveSection(self.view.getItemByIndex(0))
```

Comment:
@jrainville yes we should make `Chat` section the active one when user deletes a community which is currently the active one. For that you can just use this call: 
```
let item = self.view.model().getItemById(conf.CHAT_SECTION_ID)
self.setActiveSection(item)
```
If you decide to do that, just remove `getItemByIndex` from the model class.